CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.13.0] - released on 2017-07-12
+### Added
+ - A metadata file is generated when producing artifacts which contains metadata about components and various other information about a built artifact.
```

Comment:
Suggestion: maybe document the broad outline of what can be expected to be in the metadata file for this release?